core/indigo-core/molecule/src/molfile_loader.cpp

```diff
@@ -2907,7 +2907,10 @@ void MolfileLoader::_readCtab3000()
                     QS_DEF(Array<char>, temp_class);
                     strscan.readWord(temp_class, 0);
                     temp_class.push(0);
-                    _bmol->setTemplateAtomClass(i, temp_class.ptr());
```

Comment:
change in ket saver, not in molfile loader

Reply:
Fixed